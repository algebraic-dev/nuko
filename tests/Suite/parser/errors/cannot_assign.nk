let a: Int =
    let a = 3