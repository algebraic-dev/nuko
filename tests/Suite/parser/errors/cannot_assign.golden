• These
AssignInEndOfBlock
    ( Range
        { start = Pos
            { line = 1
            , column = 4
            }
        , end = Pos
            { line = 1
            , column = 13
            }
        }
    )

 └Program
   ├typeDecls: Empty List
   ├letDecls:
   | └List
   |   └LetDecl
   |     ├declName: (Name: ValName a 0:4-0:5)
   |     ├declArgs: Empty List
   |     ├declBody: (Lit NoExt (LInt 1:12-1:13 3))
   |     ├declRet:
   |     | └TId
   |     |   ├Local:
   |     |   | └Name: TyName Int 0:7-0:10
   |     |   └NoExt
   |     └declExt: NoExt
   ├impDecls: Empty List
   └programExt: NoExt
